onecore_maintenance_extension/hooks/__init__.py

```diff
@@ -7,7 +7,8 @@ def _update_maintenance_stages(env):
         'stage_1': {'name': 'Resurs tilldelad', 'fold': False, 'done': False, 'sequence': 2},
         'stage_3': {'name': 'Påbörjad', 'fold': False, 'done': False, 'sequence': 3},
         'stage_4': {'name': 'Väntar på beställda varor', 'fold': False, 'done': False, 'sequence': 4},
-        'stage_5': {'name': 'Avslutad', 'fold': True, 'done': True, 'sequence': 5},
+        'stage_5': {'name': 'Utförd', 'fold': False, 'done': False, 'sequence': 5},
```

Comment:
Just upgrading doesn't seem to work to make this hook run. So I guess we need to uninstall and then install? Is there a way to not do that?

Reply:
This is just for getting it correct if we ever need to reinstall the maintenance app. I've already made the changes to each environment 🙂

Reply:
So yes, to make it trigger you'd have to reinstall. Perhaps there's a "post-upgrade" hook 🤔